content/docs/get-started/aws/deploy-stack.md

```diff
@@ -55,8 +61,50 @@ Resources:
 Duration: 14s
 ```
 
-The name of the bucket that we exported is shown as a [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}).
+Remember the output you defined in the previous step? That [stack output]({{< relref "/docs/intro/concepts/stack#outputs" >}}) can be seen in the `Outputs:` section of your update. For example, the name of the bucket created above is `my-bucket-68e33ec`. To confirm your bucket has been created, let's try to list the contents of your bucket.
+
+{{< chooser language "javascript,typescript,python,go,csharp" / >}}
```

Comment:
Outside of whether to use the AWS CLI or not, I used the language chooser because the Output variable names are different between some of the languages. `bucketName`, `bucket_name`, and `BucketName`

Reply:
Looks like this still needs to be removed from the language chooser.